packages/gafl-webapp-service/src/processors/licence-type-display.js

```diff
@@ -7,10 +7,8 @@ export const licenceTypeDisplay = (permission, mssgs) => {
   // Build the display string for the licence type
   if (concessionHelper.hasJunior(permission)) {
     typesStrArr.push(mssgs.age_junior)
-  } else if (concessionHelper.hasSenior(permission)) {
-    typesStrArr.push(mssgs.over_66)
+    typesStrArr.push(', ')
```

Comment:
This should be part of the label, rather than being added as a hard-coded string